### PR TITLE
Fixed broken link blog 2022

### PR DESCRIPTION
**Pull request by niranjandarshann**
Fixed issue : Broken Link in Blog 2022  #40660

### CODE REVIEW

**Comment by mengjiao-liu** [comment]
/label tide/merge-method-squash
/lgtm


You'll need to push an empty commit to retrigger the checks on this PR, as they're currently failing.
You can run the following command.
```
git commit --amend --no-edit && git push -f
```

**Comment by niranjandarshann** [comment]
> /label tide/merge-method-squash /lgtm
> 
> You'll need to push an empty commit to retrigger the checks on this PR, as they're currently failing. You can run the following command.
> 
> ```
> git commit --amend --no-edit && git push -f
> ```

@mengjiao-liu  i run the above command and forcibly pushed it to upstream but still facing an error.

**Comment by mengjiao-liu** [comment]
Now the CI jobs has run successfully.

/lgtm

**Comment by divya-mohan0209** [comment]
This is not the right link.
/lgtm cancel

File: content/en/blog/_posts/2022-12-27-cpumanager-goes-GA.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ compatible behavior when disabled, and to document how to interact with each oth
 
 This enabled the Kubernetes project to graduate to GA the CPU Manager core component and core CPU allocation algorithms to GA,
 while also enabling a new age of experimentation in this area.
-In Kubernetes v1.26, the CPU Manager supports [three different policy options](/docs/tasks/administer-cluster/cpu-management-policies.md#static-policy-options):
+In Kubernetes v1.26, the CPU Manager supports [three different policy options](https://github.com/kubernetes/website/blob/main/content/en/docs/tasks/administer-cluster/cpu-management-policies.md#static-policy-options):
```
Comment (divya-mohan0209) [inline content/en/blog/_posts/2022-12-27-cpumanager-goes-GA.md:43]:
```suggestion
In Kubernetes v1.26, the CPU Manager supports [three different policy options](docs/tasks/administer-cluster/cpu-management-policies#static-policy-options):
```

**Comment by niranjandarshann** [comment]
pinging @divya-mohan0209  for reviewing and approval

**Review by dipesh-rawat — COMMENTED** [review]
Thanks for the fix! The new link points to the correct location.
/lgtm

Deploy Preview: [deploy-preview-40663-cpumanager-ga](https://deploy-preview-40663--kubernetes-io-main-staging.netlify.app/blog/2022/12/27/cpumanager-ga/)

**Review by divya-mohan0209 — COMMENTED** [review]
/approve

**Pull request merged**